modin/pandas/test/test_groupby.py

```diff
@@ -1811,8 +1832,11 @@ def get_columns(df):
     [pytest.param(True, marks=pytest.mark.skip("See modin issue #2513")), False],
 )
 def test_multi_column_groupby_different_partitions(
-    func_to_apply, as_index, by_length, categorical_by
+    func_to_apply, as_index, by_length, categorical_by, request
 ):
+    # Request for test parameters ids
+    _, _, _, func_to_apply_id = request.node.callspec.id.split("-")
```

Comment:
this feels like a super-hack to avoid some copy-pasting... I would rather have this passed explicitly by a parameter

Reply:
Right.

Removed this hack so now this is handled in a special comparator ([`try_almost_equals_compare`](https://github.com/dchigarev/modin/blob/e60bdb30e0ad7819f76ddefe5b8d84f2f230b42e/modin/pandas/test/utils.py#L693-L704)).